fedmsg/__init__.py

```diff
@@ -53,10 +53,13 @@ def init(**kw):
         raise ValueError("fedmsg already initialized")
 
     # Read config from CLI args and a config file
-    config = fedmsg.config.load_config([], None)
+    config = fedmsg.config.conf
 
-    # Override the defaults with whatever the user explicitly passes in.
-    config.update(kw)
+    settings = {}
+    if kw:
+        settings['settings'] = kw
+
+    config.load_config(**settings)
```

Comment:
Rather than creating a dict with a single key and immediately unpacking it, just do ``config.load_config(kw)``.

Reply:
I was waffling back and forth on how to do it. I figured there was a simpler way :). Thanks.